### PR TITLE
WIP: Remove references to yoma throughout code

### PR DESCRIPTION
**Pull request by wip-abramson**
I have removed all references to yoma in the code base. The docker containers start, but I haven't run the tests.

@morrieinmaas I couldn't see how the tests were run, you would mind running them on this branch for me?

@morrieinmaas and @TimoGlastra please review these changes - I changed a lot and not hundred percent sure everything is the way it should be. Plus this is an opportunity for us the think about the naming we use thoughout this stack.

Currently it feels like we overload the Admin term. I.e. the YOMA was an admin. But also there was a Ecosystem and Member admin. It felt confusing to me

### CODE REVIEW

**Comment by morrieinmaas** [comment]
Awesome - thanks for jumping in here @wip-abramson. I'll run the tests todayand will let you know. will also do a review

**Comment by wip-abramson** [comment]
Ah so it seems I broke something ;) 

Hopefully it isn't too opaque

**Comment by morrieinmaas** [comment]
looks like it's trying to connect to a container or url:port combo that no longer exists. Will have a look (*not* an imperative ;)).

**Review by TimoGlastra — APPROVED** [review]
This is great @wip-abramson!

**Comment by wip-abramson** [comment]
Looks like still a few bugs that need catching. Assertion miss matches etc

**Comment by morrieinmaas** [comment]
its a setup error with how the containers are starting up and there was a few name mismatches like`ecosystem.vvv` now has to be `ecosystem-partner.vvv`

**Comment by morrieinmaas** [comment]
When you run this locally you see that the governance-acapy-agent container shouts down because ```
aries_cloudagent.ledger.error.LedgerConfigError: Exception opening pool ledger default: Error: Pool timeout
  Caused by: Pool timeout
```

**Review by morrieinmaas — APPROVED** [review]
Nice job @wip-abramson 🙌. Thanks for taking the initative. I've made a few changes to fix what broke along the way.

**Comment by morrieinmaas** [comment]
@lohanspies @wip-abramson @TimoGlastra this is now passing. More comments welcome obvs otherwise feel free to merge

**Comment by wip-abramson** [comment]
Awesome @morrieinmaas! Thanks

File: .github/workflows/continuous-build.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
           echo echo waiting for dockers to startup
           while [ $i -le 2 ]
           do
-            if (docker logs $(docker ps -f name="yoma-ga-agent" | awk 'FNR == 2 {print $1}') | grep -q "::::::::::::::::::::::::::::::::::::::::::::::
+            if (docker logs $(docker ps -f name="governance-acapy-agent" | awk 'FNR == 2 {print $1}') | grep -q "::::::::::::::::::::::::::::::::::::::::::::::
             :: Alice                                    ::
```
Comment (TimoGlastra) [inline .github/workflows/continuous-build.yml:39]: I see the label `Alice` has been replaced in other places. Shouldn't this break now as it's excplitly looking for `Alice`

**Review by TimoGlastra — APPROVED** [review]
Nice, thanks @wip-abramson

File: .github/workflows/continuous-build.yml (hunk 1 above)
Comment (morrieinmaas) [reply .github/workflows/continuous-build.yml:39]: hmm fair point. Not sure why that still works then actually

**Comment by TimoGlastra** [comment]
Cam we wait to merge this PR until https://github.com/didx-xyz/aries-cloudapi-python/pull/214 is merged? I've had to resolve conflicts multiple times now and would rather do that in this name change PR

**Comment by lohanspies** [comment]
Hi @TimoGlastra when will #214 get merged so that we can merge this PR as well?

**Comment by TimoGlastra** [comment]
Working on it now... Having some really weird issues with asyncio.

**Comment by morrieinmaas** [comment]
closes in favour of #230 . Too many changes on development/main branch meanwhile.

**Pull request closed without merging**